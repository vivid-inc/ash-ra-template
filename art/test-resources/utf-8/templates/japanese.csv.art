{%

(defn csv-line
  [vals]
  (->> vals
       (map #(format "\"%s\"" %))
       (clojure.string/join ",")))

(def data [
  {"姓"       "藤沢"
   "名"       "清太郎"
   "生年月日" "1988-01-01"
   "郵便番号" "251-0036"
   "住所"     "神奈川県藤沢市江の島１丁目１－１"
   "電話番号" "050-1234-5678"}
  {"姓"       "橘"
   "名"       "涼子"
   "生年月日" "1999-01-01"
   "郵便番号" "100-0002"
   "住所"     "東京都千代田区丸の内1丁目9－1"
   "電話番号" "090-1234-5678"} ])

%}{{ (csv-line (keys (first data))) }}
{% (doseq [entry data] %}{{ (format "%s\n" (csv-line (vals entry))) }}{% ) %}
