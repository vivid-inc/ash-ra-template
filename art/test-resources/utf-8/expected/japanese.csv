"姓","名","生年月日","郵便番号","住所","電話番号"
"藤沢","清太郎","1988-01-01","251-0036","神奈川県藤沢市江の島１丁目１－１","050-1234-5678"
"橘","涼子","1999-01-01","100-0002","東京都千代田区丸の内1丁目9－1","090-1234-5678"

